dart_operators.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:io';
+
+void main(){
+    //taking input from user
+    print('Enter number a');
+    int? a = int.parse(stdin.readLineSync()!);
```

Comment:
jeżeli chcesz mieć pewność, że a będzie nie null, to możesz użyć konstrukcji z "?"

int a = int.parse(stdin.readLinseSync() ?? 0);

warto było by też obsłużyć przypadek gdy użytkownik poda np literę a nie cyfrę